src/App.tsx

```diff
@@ -1,9 +1,42 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+type Movie = {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+  imdbId: string,
+};
+
+export function filteredFilms(
+  movies: Movie[],
+  query: string,
+) {
+  let visibleFilms = [...movies];
+
+  function isIncluded(str: string): boolean {
+    return str
+      .toLocaleLowerCase()
+      .includes(query.trim().toLocaleLowerCase());
+  }
+
+  visibleFilms = visibleFilms.filter((movie) => (
+    isIncluded(movie.title) || isIncluded(movie.description)
```

Comment:
filter method returns a new array, there is no need to create a copy for the Array movies